From 8997e08c24dbaf8ad6909cf90797b5a8bdbff1dc Mon Sep 17 00:00:00 2001
From: "Ye.Li" <B37916@freescale.com>
Date: Mon, 1 Dec 2014 22:06:53 +0800
Subject: [PATCH] MLK-9941 mtd: qspi: Update device tree binding doc for
 ddrsmp property

The new property "ddrsmp" was added into device tree. Update the doc
accordingly.

Signed-off-by: Ye.Li <B37916@freescale.com>
(cherry picked from commit 4239df12c5d6c3ac19a25e120ffe17df93c358a3)
---
 Documentation/devicetree/bindings/mtd/fsl-quadspi.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/mtd/fsl-quadspi.txt b/Documentation/devicetree/bindings/mtd/fsl-quadspi.txt
index 7e1dbaf..71e620a 100644
--- a/Documentation/devicetree/bindings/mtd/fsl-quadspi.txt
+++ b/Documentation/devicetree/bindings/mtd/fsl-quadspi.txt
@@ -23,6 +23,7 @@ Optional properties:
 			      But if there are two NOR flashes connected to the
 			      bus, you should enable this property.
 			      (Please check the board's schematic.)
+  - ddrsmp: The value for DDR internal sampling point, range is 0 - 7.
 
 Part II - The DT nodes for each SPI NOR flash
 ------------------------------
-- 
1.8.0

